src/boot/ksdk1.1.0/config.h

```diff
@@ -1,44 +1,45 @@
 /*
-	Authored 2021, Phillip Stanley-Marbell.
```

Comment:
I see no reason to change the tabbing in the copyright comments.

Reply:
I believe I have updated this now.